perception/object_recognition/detection/shape_estimation/src/shape_estimator.cpp

```diff
@@ -39,20 +39,22 @@
 #include "model/yaw_fixed/bounding_box.hpp"
 #include "shape_estimation/model_interface.hpp"
 
-ShapeEstimator::ShapeEstimator()
-  : ShapeEstimator::ShapeEstimator(3, true, true){}
+ShapeEstimator::ShapeEstimator() : ShapeEstimator::ShapeEstimator(3, true, true) {}
```

Comment:
Just wondering, do you know why this constructor exists?

Reply:
I don't know. We can remove it if it is not used later.